charts/datasaur/values.yaml

```diff
@@ -160,16 +162,159 @@ jobs:
 
 loki:
   enabled: true
+  alerting:
+    contactPoints.yaml: |
+      apiVersion: 1
+
+      contactPoints:
+        - orgId: 1
+          name: cp_1
+          receivers:
+            - uid: webhook_receiver
+              type: webhook
+              settings:
+                url: https://webhook.site/8bb1d771-b0b3-4867-b680-b67471220f43
+                httpMethod: POST
+            - uid: slack_receiver
+              type: slack
+              settings:
+                url: https://hooks.slack.com/services/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
+        - orgId: 1
+          name: cp_2
+          receivers:
+            - uid: webhook_receiver_2
+              type: webhook
+              settings:
+                url: https://webhook.site/8bb1d771-b0b3-4867-b680-b67471220f43
+                httpMethod: POST
```

Comment:
what are the differences between cp_1 and cp_2?

Reply:
ini tergantung mau dikasi brp contact, cp_1 cp_2 ini cuma nama buatan, bisa diisi apa aja, ini for example aja aku kasi 2 cp @tonoman3g

Reply:
oh okei, example di sini bakalan terbuat juga gak kalau customernya ngerun helm upgrade?

Reply:
ahh, i forgot to add if statement to the config. Nanti aku buat supaya kalo g diprovide alerting yaml, g di create. Ini examplenya kyknya better aku pindah ke values.sample.yaml aja dari pada jadi default di sini yak. For the default values aku ikutin yg k ton list di #alert-grafana yak. CP nya mau dikasi placeholder apa ya? @tonoman3g

Reply:
kayanya CPnya by default kasih contoh untuk webhook aja